app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageComposerViewModel.kt

```diff
@@ -396,5 +370,35 @@ class MessageComposerViewModel @Inject constructor(
         }
     }
 
-    fun provideTempCachePath(): Path = kaliumFileSystem.rootCachePath
+    @Suppress("MagicNumber")
+    fun attachmentPicked(attachmentUri: UriAsset) = viewModelScope.launch(dispatchers.io()) {
+        val tempCachePath = kaliumFileSystem.rootCachePath
+        val assetBundle = fileManager.getAssetBundleFromUri(attachmentUri.uri, tempCachePath)
+        if (assetBundle != null) {
+            // The max limit for sending assets changes between user types. Currently, is25MB for free users, and 100MB for
```

Comment:
I wouldn't go too specific in this documentation on how big the limit is for each combination of asset vs user type. That's why we have the `GetAssetSizeLimitUseCase` for. Otherwise the moment we changed this limits in the use case, this documentation will become obsolete and we would probably forget to update it.
```suggestion
            // The max limit for sending assets changes between user and asset types. Check [GetAssetSizeLimitUseCase] class for more detailed information about the real limits. 
```